specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,91 @@
+# Numeric Format
+
+Columns that provide numeric values conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for both humans and systems.
+
+All columns capturing a numeric value, defined in the FOCUS specification, MUST be a real number represented as an integer or a decimal value as defined in this section.
+
+## Attribute ID
+
+NumericFormat
+
+## Attribute name
+
+Numeric Format
+
+## Description
+
+Rules and formatting requirements for numeric columns appearing in billing data.
+
+## Requirements
+
+* Columns with a Numeric format MUST contain a single, numeric value.
+* Numeric values MUST be expressed as integer or decimal values. Fractional notation is NOT allowed.
```

Comment:
```suggestion
* Numeric values MUST be expressed as integer or decimal values. Fractional notation MUST NOT be used.
```

Reply:
NOT is not a part of https://www.rfc-editor.org/rfc/rfc2119.txt. We can use SHALL NOT if that's preferable here.